scanner/src/main/kotlin/ScanController.kt

```diff
@@ -62,7 +62,7 @@ class ScanController(
      * A map of package [KnownProvenance]s to an [Issue] that occurred during provenance resolution for the respective
      * package.
      */
-    private val nestedProvenanceResolutionIssues = mutableMapOf<KnownProvenance, Issue>()
+    private val nestedProvenanceResolutionIssues = mutableMapOf<Identifier, Issue>()
```

Comment:
Commit message: I don't get the "it creates the resulting `Issue`s if any
specific to package(s) / IDs" part. Are maybe commas around "if any" missing?

Reply:
I've tried to further simplify the commit message a bit.